kickstarts/partials/post/python_modules.ks.erb

```diff
@@ -5,3 +5,167 @@ pip install vspk==5.3.2
 
 # For Lenovo
 pip install pylxca==2.1.1
+
+# For embedded ansible
+pip install virtualenv
+
+mkdir -p /var/lib/manageiq/
+pushd /var/lib/manageiq
+  virtualenv venv
+  source venv/bin/activate
+
+  cat > requirements.txt << EOF
+adal==1.2.1
+ansible==2.8.3
+ansible-runner==1.3.4
```

Comment:
Just caught this while doing another pass.  We probably don't wants these in here, right?

Reply:
They are there in the downstream RPM so I left them in the interest of consistency.

Reply:
Just hoping that these won't conflict with what is installed with the `ansible-runner` package in `kickstarts/partials/packages/includes.ks.erb`

Reply:
It won't. This will install binstubs in the venv directory. So we won't get this one unless we were to activate the venv before running `ansible-runner` (which we wont).

In fact we will never actually activate the venv. We are only passing the internal directory as a part of the `PYTHONPATH` so the stubs for these modules never actually get on our `PATH` when we're executing runner.

To illustrate this is from the appliance I've been doing the testing for this PR on:
```
[root@localhost venv]# pwd
/var/lib/manageiq/venv
[root@localhost venv]# which ansible-runner
/usr/bin/ansible-runner
[root@localhost venv]# source bin/activate
(venv) [root@localhost venv]# which ansible-runner
/var/lib/manageiq/venv/bin/ansible-runner
(venv) [root@localhost venv]# deactivate
[root@localhost venv]# which ansible-runner
/usr/bin/ansible-runner
```

Reply:
Plus if this is a problem here then it will be a problem in CF as well so it's probably worth dealing with it in both places if it is an issue. As you pointed out in https://github.com/ManageIQ/manageiq-appliance-build/pull/341#issuecomment-519687490 :trollface:

Reply:
I think this is resolved now that we don't have them in the list after using `pip freeze -l`

Reply:
> Plus if this is a problem here then it will be a problem in CF as well so it's probably worth dealing with it in both places if it is an issue. As you pointed out in #341 (comment) :trollface:

So... you are saying you believe me now :trollface: